### DISCUSSION TITLE
How to add target attr in react pdf Link

### DISCUSSION THREAD

**Discussion by poshan-kandid** [diegomura/react-pdf#1714] [Q&A]
Can we add target prop in react pdf Link.
Like this =>
"`<Link src="https://www.google.com" target="_blank">Open</Link>`"

**Comment by nalliot** [comment]
It appears it is not possible as it is. The `setLink` method is not setting the props.target in anyway so we cannot set target. 

Please note that while target is not a standard pdf link attribute, it is interpreted by most browser built-in pdf viewers in 2023.
